src/components/container/SeverityCalendar/SeverityCalendar.tsx

```diff
@@ -0,0 +1,158 @@
+                    let exists = resultByDateMap.get(key);
+                    if (exists) {
+                        if ((dateOfSeverity > exists.dateObserved) ||
+                            (dateOfSeverity === exists.dateObserved && grouping.severityAnswer.date > exists.dateEntered)) {
+                            exists.dateObserved = dateOfSeverity;
+                            exists.severity = severity;
+                            exists.dateEntered = grouping.severityAnswer.date;
+                        }
+                    } else {
+                        resultByDateMap.set(key, { dateObserved: dateOfSeverity, severity: severity, dateEntered: grouping.severityAnswer.date });
+                    }
+                }
+            }
+        })
+
+        setData(resultByDateMap);
+    }
+
+    function groupAnswersByResult(results: SurveyAnswer[]) {
+        var groupedByResult = new Map<Guid, { dateObservedAnswer?: SurveyAnswer, severityAnswer?: SurveyAnswer }>();
+
+        results.forEach((r) => {
+            var exists = groupedByResult.get(r.surveyResultID);
+            var completeResult = exists ?? { dateObservedAnswer: undefined, severityAnswer: undefined };
+            if (r.answers && r.answers[0] !== "") {
+                if (r.resultIdentifier === props.severityResultIdentifier) {
+                    completeResult.severityAnswer = r;
+                }
+                else if (r.resultIdentifier === props.dateRecordedResultIdentifier) {
+                    completeResult.dateObservedAnswer = r;
+                }
+
+                groupedByResult.set(r.surveyResultID, completeResult);
+            }
+        })
+
+        return groupedByResult;
+    }
+
+    const stateConfiguration: CalendarDayStateConfiguration = {
+        'mild': {
+            style: {
+                background: '#F7DA19'
+            },
+            streak: true,
+            streakColor: '#FDEF97'
+        },
+        'moderate': {
+            style: {
+                background: 'var(--mdhui-color-warning)'
+            },
+            streak: true,
+            streakColor: '#FCCCC7'
+        },
+        'severe': {
+            style: {
+                background: 'var(--mdhui-color-danger)'
+            },
+            streak: true,
+            streakColor: '#FCCCC7'
+        },
+    };
+
+    const computeStateForDay = (date: Date): string => {
+        let key: string = format(startOfDay(date), 'MM/dd/yyyy');
+        return data?.get(key)?.severity ?? "";
+    };
+
+    const renderDay = (year: number, month: number, day?: number): React.JSX.Element => {
+        return <CalendarDay
+            year={year}
+            month={month}
+            day={day}
+            stateConfiguration={stateConfiguration}
+            computeStateForDay={computeStateForDay}
+        />;
+    };
+
+    useInitializeView(initialize, [], [props.previewState]);
+
+    if (!data) {
+        return <LoadingIndicator />
+    }
+    else {
```

Comment:
Minor:  My settings may be the ones that are off, but I think we usually like the `else {` to be on the same line as the `}`.

Not a big deal really, but I think we should try to all sync up on the settings so we can avoid diffs when we just autoformat files.

Reply:
We slacked and agreed to use } else {